kafka-bare-quickstart/pom.xml

```diff
@@ -46,6 +46,12 @@
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-kafka-client</artifactId>
     </dependency>
+    <dependency>
+      <!-- Workaround for https://github.com/quarkusio/quarkus/issues/22000 TODO: should be removed, when quarkus team will fix the issue -->
+      <groupId>org.apache.kafka</groupId>
+      <artifactId>kafka-clients</artifactId>
+      <version>2.8.1</version>
```

Comment:
thanks! 
I'm opening a PR against the bindings to avoid pulling 2.6.0: https://github.com/smallrye/smallrye-mutiny-vertx-bindings/pull/453